Makefile

```diff
@@ -18,7 +18,7 @@ checkrst:
 	# page properly.
 	python setup.py check --restructuredtext
 
-test: .install .install-test-deps flake8
+test: clean .install .install-test-deps flake8
```

Comment:
I assume this is the main contribution to fixing the problem, right?

Reply:
Yes exactly.